app/package.json

```diff
@@ -25,7 +25,7 @@
     "codemirror-mode-elixir": "1.1.1",
     "deep-equal": "^1.0.1",
     "dexie": "^2.0.0",
-    "dugite": "1.58.0",
+    "dugite": "1.60.0",
```

Comment:
This is important because `dugite` now has ARM64 support, correct? If so, we should run though the various Git operations in the app as part of the review.

Reply:
@shiftkey yes, I had to upgrade dugite to 1.60.0 for arm64 support.